crates/common/src/prov/model/from_json_ld.rs

```diff
@@ -54,7 +54,7 @@ impl Loader<IriBuf, ()> for ContextLoader {
             let json = json!({
              "@context": crate::context::PROV.clone()
             });
-            let value = json_ld::syntax::Value::from_serde_json(json, |_| ());
+            let value = json_syntax::Value::from_serde_json(json, |_| ());
```

Comment:
Is this why we now depend on `json-syntax`? Odd it's for just this `from_serde_json` usage, there's a couple more.